README.md

```diff
@@ -586,13 +586,14 @@ The IP configuration supports the following options:
 
   Static route configuration can be specified via a list of routes given in the
   `route` option. The default value is an empty list. Each route is a dictionary with
-  the following entries: `network`, `prefix`, `gateway`, `metric` and `table`.
-  `network` and `prefix` specify the destination network. `table` supports both the
-  numeric table and named table. In order to specify the named table, the users have
-  to ensure the named table is properly defined in `/etc/iproute2/rt_tables` or
-  `/etc/iproute2/rt_tables.d/*.conf`.
-  Note that Classless inter-domain routing (CIDR) notation or network mask notation
-  are not supported yet.
+  the following entries: `network`, `prefix`, `gateway`, `metric`, `table` and
+  `route_type`. `network` and `prefix` specify the destination network. `table`
+  supports both the numeric table and named table. In order to specify the named
+  table, the users have to ensure the named table is properly defined in
+  `/etc/iproute2/rt_tables` or `/etc/iproute2/rt_tables.d/*.conf`. And the supported
+  `route_type` are `blackhole`, `prohibit`, `unreachable` and these `route_type` routes
+  can not have next hop (gateway) specified. Note that Classless inter-domain routing
```

Comment:
Please re-rephrase this to not start a new sentence with "and". The documentation needs to clarify that it is also possible not to specify a route_type. Also, it seems that it should be called just "type" instead of "route_type", since it is already clear that the properties are about routes. It would also be nice to link to the type definitions at ip-route here - or some other good source.

Reply:
I re-phrased the text and attached the link for the iproute man page.